src/views/customer/CustomerListView/index.tsx

```diff
@@ -14,16 +14,35 @@ const useStyles = makeStyles((theme: any) => ({
   }
 }));
 
+const client =  createClient({
+  url: 'https://iteria-app-example01.herokuapp.com/v1/graphql',
+  fetchOptions: () => {
+    return {
+      headers: {
+        'x-hasura-admin-secret' : 'iteria-app-example01-421769',
+        'Content-Type': 'application/json',
+        'Accept': 'application/json',
```

Comment:
should not be necessary to provide json headers (they are reasonable defaults...)